packages/fields/src/types/OEmbed/Implementation.js

```diff
@@ -215,7 +216,7 @@ export class OEmbed extends Implementation {
     // Convert them into a more GraphQL friendly format
     return {
       originalUrl: inputUrl,
-      type,
+      type: type || 'link',
```

Comment:
```suggestion
      type: type || 'url',
```

Slightly more accurate I think? We're not really storing/returning a "link" -- links are things rendered in HTML documents. We're storing the URL that a link might use.

Reply:
`link` is a specific type name used by the OEmbed spec, and matches a specific GraphQL type defined above. `url` in this case may be more accurate, but it's of type "OEmbed Link"